src/scss/06_components/layout/_component.scss

```diff
@@ -6,8 +6,8 @@
 
 // When components are full width and do not contain liftup-with-img-bg or content-cards,
 // they should have padding that keeps them from touching side.
-.components--lower > .component:not(.component--liftup-with-image-bg, .component--content-cards),
-.page-type--landing-page.no-sidebar .components--upper > .component:not(.component--liftup-with-image-bg, .component--content-cards) {
+.components--lower > .component:not(.component--liftup-with-image-bg, .component--content-cards-grey),
```

Comment:
This is because the content cards without gray background were actually touching the edges of the lower content region on smaller screens. This should fix that :) The screenshot shows what happens if they are left as is.

<img width="394" alt="Screenshot 2024-08-29 at 15 05 06" src="https://github.com/user-attachments/assets/b3f0c689-8c54-4faf-a1b6-91f22d81edfe">